Refresh the workshop - 2022 edition (#19)
Particular/Workshop.NServiceBus: README.md

### Install the pre-requisites

To complete the exercises, you require a Windows machine and Visual Studio.

#### Visual Studio

Install [Visual Studio 2019](https://www.visualstudio.com) or later (Community, Professional, or Enterprise) with the following workloads:

- .NET desktop development
- ASP.NET and web development

The following frameworks need to be installed:

- .NET Framework 4.6.1
- .NET 6

#### Particular Software Platform

The Particular Service Platform includes ServiceControl, ServicePulse and ServiceInsight.
The samples include a platform connection package that will fire up both ServiceControl and ServicePulse, without requiring any installation.

> In order to try out ServiceInsight, an installation is required. The latest version can be downloaded [on our website](https://github.com/Particular/ServiceInsight/releases/tag/2.9.2). Once installed, ServiceInsight needs to be connected to ServiceControl. 

Reviewer: ```suggestion
In order to try out ServiceInsight, an installation is required. The latest version can be downloaded [from GitHub](https://github.com/Particular/ServiceInsight/releases/latest). Once installed, ServiceInsight needs to be connected to ServiceControl. 
```
